### DISCUSSION TITLE
Origin of OMR address on child device

### DISCUSSION THREAD

**Discussion by dt-atmosic** [openthread/openthread#9463] [Q&A] [answered]
Question about this part of section 4 in https://openthread.io/codelabs/openthread-border-router#3

```
> ipaddr
fd76:a5d1:fcb0:1707:d3dc:26d3:f70b:b927
fda8:5ce9:df1e:6620:0:ff:fe00:4001
fda8:5ce9:df1e:6620:ed74:123:cc5d:74ba
fe80:0:0:0:d4a9:39a0:abce:b02e
Done
``` 

The `fd76:a5d1:fcb0:1707:d3dc:26d3:f70b:b927` address corresponds to the `fd76:a5d1:fcb0:1707::/64` OMR prefix in the netdata that was published by the border router.  Where in the openthread source code is that IPv6 address added to the child?

I'm trying to reproduce the steps in that guide, and everything up to that point looks good, but I do not see this OMR address appearing on the child device.

**Comment by jwhui** [comment, marked as answer]
SLAAC addresses are configured here:
https://github.com/openthread/openthread/blob/0c5966de817afa3c42917e26d96ee2dcb6d4b80f/src/core/utils/slaac_address.cpp#L242

**Reply by dt-atmosic** [reply]
Perfect - thank you so much!  My device was missing OPENTHREAD_CONFIG_IP6_SLAAC_ENABLE.
